dcicutils/s3_utils.py

```diff
@@ -220,8 +225,9 @@ def unzip_s3_to_s3(self, zipped_s3key, dest_dir, acl=None):
                 # files twice that we want to further interogate
                 the_file = zipstream.open(file_name, 'r').read()
                 file_to_find = file_name.split('/')[-1]
-                ret_files[file_to_find] = {'s3key': s3_key,
-                                           'data': the_file}
+                if not do_not_store:
+                    ret_files[file_to_find] = {'s3key': s3_key,
+                                               'data': the_file}
                 self.s3_put(the_file, s3_file_name, acl=acl)
```

Comment:
Shouldn't this be indented into the `if`? It looks like if you set this variable it just lies about what it did.

Reply:
What do you mean? Yes, it is indented into the `if`.